README.md

```diff
@@ -152,6 +150,20 @@ CLOUDANT_PASSWORD=<password>
 **Note**: We recommend using [IAM](#iam-authentication) for Cloudant and
 [Session](#session-cookie-authentication) for CouchDB authentication.
 
+#### Authenticate with external configuration or the programmatic way
+
+There are other ways to authenticate rather than setting environment variables:
```

Comment:
You can remove this and the line below.  The paragraphs below are enough and provide the two auth options.